test/tests/example.js

```diff
@@ -1,46 +1,52 @@
-describe("Simple tests examples", function() {
-  it("should detect true", function() {
+describe("Simple tests examples", function () {
```

Comment:
Thank you for your feedback. Actually it is my fault. I just wanted to test if the karma update was successful. I shouldn't commit the change on the example tests at all. I will roll back the changes on this file.